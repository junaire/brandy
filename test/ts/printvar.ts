var x = 5n;
console.log(x);
