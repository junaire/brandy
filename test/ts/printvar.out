@main {
  v0: int = const 5;
  x: int = id v0;
  v1: int = id x;
  print v1;
  v2: int = const 0;
}
